### ISSUE TITLE
Plugin request - Avatars in roster and MUC participant list

### ISSUE THREAD

**Issue by muppeth** [conversejs/community-plugins#10]
I see there is such feature on Pade, I was wondering, if it would be easy for you @deleolajide to port it to converse. I love looks of Pade, including the default when no avatar is given (there was also a random color assigned to them right?).

**Comment by deleolajide** [comment]
> I was wondering, if it would be easy for you @deleolajide to port it to converse.

The Pàdé look was created out of an experiment on this [converse issue](https://github.com/conversejs/converse.js/issues/1349). It is mostly a direct hack to the converse.js code and default templates. I should either upstream it back to Converse with the accompanying settings or re-implement as a plugin and override the templates, but I am very pre-occupied with other things right now.

I may revisit it when I have to tackle the imminent upgrading of Pàdé to Converse 5.

**Comment by muppeth** [comment]
I see. It would explain why i couldnt find it in the plugin section of pade. Is there a place (excuse me if its obvious and i missed it) where you have the source code of the converse. As far as I can see there is only dist builds on Pade's repo.

**Comment by deleolajide** [comment]
Sorry, I am old school. I work directly with HTML, JS and CSS files. That is my source code :-)
My hacking can be identified with the // BAO comments

**Comment by muppeth** [comment]
Damn..... Taht would explain why I couldnt find it anywhere in commit tree. So you hacked directly on dist files or you have somewhere templates/css/js files you touched.

**Comment by muppeth** [comment]
BTW is there any Pade instance with public signup so that one could check all the hardcoded features. checking the commented changes, I see there is quite some of them (reactions, drag and drop file upload etc).

**Comment by deleolajide** [comment]
http://pade.chat runs Pàdé in web and pwa mode. Please note that pwa mode is not yet optimised, but it works. You can create an account and play with most features as the openfire server has all the required plugins installed. 

To get the best results, use the chrome extension and point at pade.chat:443 as server and pade.chat as domain.

**Comment by deleolajide** [comment]
> I guess this is it: https://github.com/deleolajide/converse.js (if there are specific commits to point me to would be amazing)

No. This - https://github.com/igniterealtime/Pade/blob/master/extension/inverse/dist/converse.js

**Comment by muppeth** [comment]
> http://pade.chat runs Pàdé in web and pwa mode.
Thanks. Will do

> No. This - https://github.com/igniterealtime/Pade/blob/master/extension/inverse/dist/converse.js
Yeah I figured. But was hoping you have somewhere the sources before they were build (easier to check which exactly template or css file was touched. 
But I guess you hacked directly on dist files so will need to sieve throurgh. 

Thanls for all the info. I made account on pade.chat and damn this is packed with awesome features. You should definitely do it new school and use git so its easier for leachers like me to get all the good stuff :P
